src/tests/PaginationArrow.spec.js

```diff
@@ -34,31 +32,10 @@ describe('The Pagination Arrow component', function () {
       onTrigger: clickTrigger,
     });
     expect(this.renderedComponent).toEqual(
-      <div className='my-class' onClick={clickTrigger}>
-        <div className='PaginationArrowIcon' />
+      <div bemBlock='DateRangePicker' className='DateRangePicker__PaginationArrow DateRangePicker__PaginationArrow--next' onClick={clickTrigger} >
+        <div className='DateRangePicker__PaginationArrowIcon DateRangePicker__PaginationArrowIcon--next' />
       </div>
     );
   });
 
```

Comment:
But I like whitespaces at the end of self-closing XML tags :) Is there a way to set-up a linting rule for this?